dramatiq/brokers/rabbitmq.py

```diff
@@ -246,8 +246,9 @@ def declare_queue(self, queue_name, *, ensure=False):
             self._ensure_queue(queue_name)
 
     def _ensure_queue(self, queue_name):
-        attempts = 1
+        attempts = 0
```

Comment:
It seems like just changing the initialization to `0` would be enough, no? I don't think we have to move the increment to the top of the loop, but maybe I'm missing something. Ditto for the other snippet.

Reply:
I agree, but thought it’s easier to read with incrementing at the beginning of the loop instead of the error only. However, that also _adds_ an unnecessary increment in the success case.

I’m easy either way. Want me to move the increment back down?

Reply:
Yeah, let's move it back down. I like having it co-located with the check. Thanks!